l10n_ch_import_winbiz/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module add the ability to import salary XML file from Winbiz software (www.winbiz.ch) 
+
+Configuration
+=============
+
+
+
+Usage
+=====
+In order to import your Winbiz Salary.txt 
+file you must complete the following requirements : 
```

Comment:
s/requirements : /requirements/

Plus add a line break or your list won't be well displayed